src/lucky_web/log_handler.cr

```diff
@@ -40,4 +44,12 @@ class LuckyWeb::LogHandler
 
     "#{(millis * 1000).round(2)}µs"
   end
+
+  private def timestamp(time)
+    if settings.show_timestamps && time
```

Comment:
@paulcsmith compiler complained without checking that `time` is set. Not sure whether best choice is this conditional or `time ||= Time.now` before so it always logs timestamp, even if it's maybe bit off. It would probably help with parsing if timestamp is always present so format is consistent?

Reply:
Ah I see. The compiler caught a potential bug. The `time` might be `nil` because it could raise and therefore `time` may not have been set. I think what we should do is something like this:

```crystal
This way if `time` is nil it is set to Time.now
private def timestamps(time : Time = Time.now)
```

Like you said it may be a bit off, but I think that's better than not having any timestamp

Reply:
Fixed, default value didn't work, but `time || Time.now` did